internal/app/cache/cache.go

```diff
@@ -110,7 +110,7 @@ func (c *cache) Fetch(key string) (*Resource, error) {
 func (c *cache) SetResponse(key string, resp v2.DiscoveryResponse) ([]*v2.DiscoveryRequest, error) {
 	c.cacheMu.Lock()
 	defer c.cacheMu.Unlock()
-	marshaledResources, err := marshalResources(resp.Resources)
+	marshaledResources, err := MarshalResources(resp.Resources)
```

Comment:
Wondering why this needs to be public.

Reply:
This is used by the orchestrator tests in order to compare the marshaled response with the mock desired one: https://github.com/envoyproxy/xds-relay/pull/53/files#diff-2c2ed0f611d2764f6a076c08ee2b18f7R88. Open to suggestions on how to better do this!

Reply:
No worries. This makes sense. 👍